src/test/java/Homework16.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework16 extends BaseTest {
+    @Test
+    public static void RegistrationNavigation() {
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221780546-493e3d9d-e21e-4fe0-9611-78d40bba9ccd.png)

Reply:
I'm unable to follow what the problem is, please help.